packages/completer/src/widget.ts

```diff
@@ -439,6 +439,8 @@ export class Completer extends Widget {
         if (populated) {
           this.update();
         }
+
+        this._cycle(event.shiftKey ? 'up' : 'down');
```

Comment:
I am not sure if having discordant keybindings is a good thing, this is:
- <kbd>shift</kbd> + <kbd>Tab</kbd> means "go up", but
- <kbd>Tab</kbd> meaning "insert active completion item"
might end up confusing users and lead to frustration ("it works one way, why doesn't it work the other way too!").

Would it be possibly better to keep only arrows for cycling for this PR, and in another PR add a capability to change the keybindings from "arrows for cycling and Tab for insert" to "Tab/Shift Tab for cycling and Enter for insert"?

Reply:
Nevermind I misunderstood it.